XCUITests/SyncFAUITests.swift

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class SyncUITests: BaseTestCase {
+    func testUIFromSettings () {
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"])
+        XCTAssertTrue(app.navigationBars["Client.FxAContentView"].exists)
+        XCTAssertTrue(app.webViews.textFields["Email"].exists)
+        XCTAssertTrue(app.webViews.secureTextFields["Password"].exists)
+        XCTAssertTrue(app.webViews.buttons["Sign in"].exists)
+    }
+
+    func testPlaceholderValues () {
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"])
+        let mailPlaceholder = "Email"
+        let passwordPlaceholder = "Password"
+
+        let defaultMailPlaceholder = app.webViews.textFields["Email"].placeholderValue!
+        let defaultPasswordPlaceholder = app.webViews.secureTextFields["Password"].placeholderValue!
+        XCTAssertTrue(mailPlaceholder == defaultMailPlaceholder, "The mail placeholder does not show the correct value")
+        XCTAssertTrue(passwordPlaceholder == defaultPasswordPlaceholder, "The password placeholder does not show the correct value")
+    }
+
+    func testTypeOnGivenFields() {
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"])
+
+        // Tap Sign in without any value in email Password focus on Email
+        navigator.performAction(Action.TapOnSignInButton)
+        waitforExistence(app.webViews.staticTexts["Valid email required"])
+
+        // Enter only email, wrong and correct and tap sign in
+        navigator.performAction(Action.TypeInvalidEmail)
+        navigator.performAction(Action.TapOnSignInButton)
+        waitforExistence(app.webViews.staticTexts["Valid email required"])
+
+        navigator.performAction(Action.TypeCorrectEmail)
+        navigator.performAction(Action.TapOnSignInButton)
+        waitforExistence(app.webViews.staticTexts["Valid password required"])
+
+        // Enter invalid (too short) and incorrect password
+        navigator.performAction(Action.TypeInvalidPassword)
+        navigator.performAction(Action.TapOnSignInButton)
+        waitforExistence(app.webViews.staticTexts["Must be at least 8 characters"])
+
+        navigator.performAction(Action.TypeIncorrectPassword)
+        navigator.performAction(Action.TapOnSignInButton)
+        waitforExistence(app.webViews.staticTexts["Unknown account."])
+        XCTAssertTrue(app.webViews.links["Sign up"].exists)
+    }
+
+    func testCreateAnAccountLink() {
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.links["Create an account"])
```

Comment:
Shouldn't this test be in `testUIFromSettings`?

Reply:
Well, this is not an option in the Setting list. The idea of this test is to check that from the FxSignin Screen is possible to create a FxAccount. I added it here because it is where we are checking the FxSignin screen, but I can move it to the other test suite if that sounds better for you

Reply:
I mention this test because it goes to `FxASigninScreen` which has the Create an account link. 

This is a trivial thing, I'm not digging in my heals :-)